Selenium/selenium10.rb

```diff
@@ -0,0 +1,33 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path = 'C:\Users\neera.yadav\Documents\BrowserDrivers\chromedriver-win64\chromedriver-win64\chromedriver.exe'
+
+driver = Selenium::WebDriver.for :chrome
+
+driver.get 'https://computer-database.gatling.io/computers'
+
+target_column_header_text = "Computer name"
+
+header_row = driver.find_element(css: 'thead tr')
+
+target_column_index = header_row.find_elements(tag_name: 'th').find_index do |header_cell|
+  header_cell.text == target_column_header_text
+end
```

Comment:
You do not need to fetch the header row separately. You only need to read the `tr` elements of the table which will also fetch the header values. as they too are enclosed within a `tr` tag.

Reply:
Done